### DISCUSSION TITLE
Deep transform_keys

### DISCUSSION THREAD

**Discussion by mjkaufer** [ElMassimo/oj_serializers#32] [Q&A]
If I have some nested data (i.e. posts -> user), I don't think there's currently a way to recursively apply `transform_keys :camelize` without writing a serializer for each sub-component (which can be tedious in more complex data models)

Are there plans to add a version of `transform_keys :camelize` that applies to every nested key of arbitrary depth?

**Comment by ElMassimo** [comment]
Hi Matthew!

Not currently, as being able to statically generate the transformed key is the only way to guarantee high performance.
